Library/Homebrew/utils/formatter.rb

```diff
@@ -0,0 +1,46 @@
+require "utils/tty"
+
+module Formatter
+  module_function
+
+  def arrow(string, color: nil)
+    prefix("==>", string, color)
+  end
+
+  def headline(string, color: nil)
+    arrow("#{Tty.bold}#{string}#{Tty.reset}", color: color)
+  end
+
+  def success(string, label: nil)
+    label(label, string, :green)
+  end
+
+  def warning(string, label: nil)
+    label(label, string, :yellow)
+  end
+
+  def error(string, label: nil)
```

Comment:
Without a label, the message will be styled the same as the label

`Formatter.error("This string is red.)`

`Formatter.error("This message is not red.", label: "Red Label")`.